Docs/development/jasp-build-guide-macos.md

```diff
@@ -33,25 +32,43 @@ You can download and install Homebrew using the following command. After running
+Download the [Qt Installer](https://www.qt.io/download), and run the installer. You may need to create a [Qt Account](https://login.qt.io/login) before you will be able to download and install the Qt Framework.
 
-JASP's CMake configuration file provides several variables that allow you to customize your build. Here, we mention a few important ones, and you can find the rest of the variables for further customization in your "Projects" tab.
+Follow the installation steps, and select the following items from the list of components in the "Select Components" step:
+
+- **Qt**
+	- **Qt 6.2.4** (or newest stable Qt)
+		- [x] macOS
+		- [x] Qt 5 Compatibility Module
+		- **Additional Libraries**
+			- [x] Qt Web Engine
+			- [x] Qt Web Channel
+			- [x] Qt WebSockets
+	- **Developer and Designer Tools**
+		- [x] Qt Creator 7
+		- [x] Qt Creator 7 Debug Symbols
+		- [x] CMake
+		- [x] Ninja
+
+### Configuring and Building JASP
+
+After installing all your dependencies, go to the Qt Creator app - go to open - your cloned `jasp-desktop` folder and open `CMakeLists.txt`. By doing so, Qt Creator configures the JASP project using CMake. 
+
+JASP's CMake configuration file provides several variables that allow you to customise your build. Here, we mention a few important ones, and you can find the rest of the variables for further customization in your "Projects" tab.
 
 - `GITHUB_PAT`
-	- If you don't have a `GITHUB_PAT`, you can create one by following this instruction: [GitHub Personal Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
+	- If you don't have a `GITHUB_PAT`, you can create one by following these instructions: [GitHub Personal Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token).
 	- After creating a new PAT, you can set that `GITHUB_PAT` variable in two ways,
 		- If you are using the Qt Creator, you need to either set the `GITHUB_PAT` directly to your CMake variables, under the "Projects->Build->Current Configuration". Search the list for `GITHUB_PAT`, and paste your PAT value there. This will instruct the CMake to use your PAT whenever necessary. 
 	- If you are using the command line, CMake looks for this variable in your environment variable and is able to automatically find and use it, if your `GITHUB_PAT` can be found in your environment variables.
-		- Adding this like to your `.bash_profile`, or `.zshrc` will fasciliate this process, `export GITHUB_PAT=your github pat`
-- `INSTALL_R_MODULES`
-	- If you wish to install JASP modules, you need to toggle this option `ON`. In Qt Creator->Projects->Builds->Current Configuration, you can find this variable, and check the checkbox. Or if you are using a command line, you can include this in your command line by `-DINSTALL_R_MODULES=ON`.
```

Comment:
@amirmasoudabdol did you remove this? Or did something go wrong with me pulling and pushing?

Reply:
Oh, sorry my bad, I only wanted to remove the part about `INSTALL_R_MODULES`. I can fix it...

Reply:
It's ok apparently. The part about `INSTALL_R_MODULES` isn't necessarily because it's on by default. That's it.